Src/BootCamp.Chapter/Checks.cs

```diff
@@ -1,39 +1,215 @@
-        }
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            string richestName = "";
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var account = new Account(MakeStuffWithInput.GetNameFromString(peopleAndBalances[i]), MakeStuffWithInput.GetBalanceFromString(peopleAndBalances[i]));
 
+                if (i == 0)
+                {
+                    richestName = account.GetName();
+                    amount = account.CurrentBalance();
+                }
+                else if (amount == account.CurrentBalance())
+                {
+                    if (personCount == 1)
+                    {
+                        richestName += " and " + account.GetName();
+                        personCount++;
+                    }
+                    else
+                    {
+                        richestName = richestName.Replace(" and ", ", ");
+                        richestName += " and " + account.GetName();
+                        personCount++;
+                    }
+                }
+                else if (amount < account.CurrentBalance())
+                {
+                    richestName = account.GetName();
+                    amount = account.CurrentBalance();
+                    personCount = 1;
+                }
+            }
+
+            return MakeStuffWithInput.MakefullstringRichestPeople(personCount, richestName, amount);
+        }
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            string biggestLossName = "";
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var account = new Account(MakeStuffWithInput.GetNameFromString(peopleAndBalances[i]), MakeStuffWithInput.GetBalanceFromString(peopleAndBalances[i]));
+
+                if (account.MoreThan1Balance())
+                {
+
+                }
+                else if (biggestLossName == "")
+                {
+                    biggestLossName = account.GetName();
+                    amount = account.BiggestLoss();
+                }
+                else if (amount == account.BiggestLoss())
+                {
+                    if (personCount == 1)
+                    {
+                        biggestLossName += " and " + account.GetName();
+                        personCount++;
+                    }
+                    else
+                    {
+                        biggestLossName = biggestLossName.Replace(" and ", ", ");
+                        biggestLossName += " and " + account.GetName();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)